.ci/Jenkinsfile

```diff
@@ -132,15 +135,18 @@ pipeline {
               sh(label: 'set permissions', script: '''
                 chmod -R ugo+rw examples
               ''')
-              dir("__tests__/e2e"){
-                timeout(time: 10, unit: 'MINUTES') {
-                  sh(label: 'run e2e tests',script: 'npm run test')
-                }
+              dir("${E2E_FOLDER}"){
+                sh(label: 'run e2e tests',script: 'npm run test')
               }
             }
           }
         }
       }
+      post {
+        always {
+          junit(allowEmptyResults: true, keepLongStdio: true, testResults: "${BASE_DIR}/${E2E_FOLDER}/**/junit.xml")
+        }
+      }
```

Comment:
It works now, though there were test failures

- https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-rum%2Felastic-synthetics/detail/PR-235/3/tests

![image](https://user-images.githubusercontent.com/2871786/109802962-5dd17700-7c18-11eb-9079-c812d3a36ec0.png)

Reply:
@vigneshshanmugam shoudn't stack trace be more verbose? 
![image](https://user-images.githubusercontent.com/3505601/109803724-af323400-7c21-11eb-87ac-538b619771c0.png)